ramalama/common.py

```
@@ -774,3 +775,26 @@ def __str__(self):
+    parent is found.
+
+    Args:
+        args: CLI arguments containing the store path.
+
+    Returns:
+        A dictionary with disk usage stats  - None for all stats if no path is found.
+    """
+    current_path = Path(args.store)
+    while True:
+        try:
+            return dict(zip(("total", "used", "free"), shutil.disk_usage(current_path)))
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)